src/main/java/frc/robot/Limelight.java

```diff
@@ -22,4 +61,168 @@ public void togglePipeline() {
         }
 
     }
+
+    /*
+     * Gets the horizontal offset from the limelight to the target
+     * Outputs in degrees because it sees how far to the left or right the camera is
+     */
+    public double getHorizontalOffset() {
+        
+        return table.getEntry("tx").getNumber(0).doubleValue();
+
+    }
+
+    /*
+     * Returns the vertical offset angle from the limelight to the target
+     */
+    public double getVerticalOffset() {
+
+        return table.getEntry("tx").getNumber(0).doubleValue();
+
+    }
+
+    /*
+     * Returns the skew of the robot
+     * Outputs between -90 and 0 degrees
+     */
+    public double getSkew() {
+
+        return table.getEntry("ts0").getNumber(0).doubleValue();
+
+    }
+
+    /*
+     * Returns the total area that the current target takes up on the limelight's screen
+     * Outputs a value associated with the percent of the screen being taken up
+     */
+    public double getTargetArea() {
+
+        return table.getEntry("ta").getNumber(0).doubleValue();
+
+    }
```

Comment:
add some kind of error handling or target check before returning because 0 is a possible valid value for these variables